lambdas/constants.js

```diff
@@ -42,7 +42,9 @@ const ReadOnlyPolicy = `{
               "sns:GetSubscriptionAttributes",
               "ssm:ListCommandInvocations",
               "ce:GetCostAndUsage",
-              "redshift-serverless:List*"
+              "redshift-serverless:List*",
+              "lambda:GetCodeSigningConfig",
+			  "lambda:GetFunctionCodeSigningConfig"
```

Comment:
@rgurav-uptycs we need to update zip file 
and keep it in proper format

Reply:
@llukalapu-uptycs How we can do that?

Reply:
@llukalapu-uptycs fixed the formatting

Reply:
@llukalapu-uptycs How can we update the zip file

Reply:
Done.